src/views/Package/Package.tsx

```diff
@@ -53,15 +84,7 @@ export const Package: FunctionComponent = () => {
         />
       </Box>
       {/* Readme and Api Reference Area */}
-      {markdownResponse.data &&
-        !markdownResponse.loading &&
-        assemblyResponse.data &&
-        !assemblyResponse.loading && (
-          <PackageDocs
-            assembly={assemblyResponse.data}
-            markdown={markdownResponse.data}
-          />
-        )}
+      {docsOrError(markdownResponse, assemblyResponse, language)}
```

Comment:
I would prefer to see this either as a function component - `<DocsOrError {...props} />` or as 
```tsx
{hasDocs ? (
  <PackageDocs {...props} />
) : (
  <ErrorMessage {...props} />
)}
```

So that it is consistent with how we define & call our function components

Reply:
Done